server/src/test/java/org/candlepin/test/DatabaseTestFixture.java

```diff
@@ -507,4 +544,114 @@ private static void insertValidationEventListeners(Injector inj) {
         registry.getEventListenerGroup(EventType.PRE_UPDATE).appendListener(listenerProvider.get());
         registry.getEventListenerGroup(EventType.PRE_DELETE).appendListener(listenerProvider.get());
     }
+
+    protected Set<String> typeLabels(Set<String> types) {
+        return types;
+    }
+
+    protected List<String> skus(List<String> skus) {
+        return skus;
+    }
+
+    protected List<String> subscriptionIds(List<String> subscriptionIds) {
+        return subscriptionIds;
+    }
+
+    protected List<String> contracts(List<String> contracts) {
+        return contracts;
+    }
+
+    protected String ownerKey(String key) {
```

Comment:
I was rethinking this and it should really not be in superclass, but what if I had moved these to ConsumerCuratorTest, would have you still disagreed? Consider count of lines when every unit test must have declared null variables, it's more verbose.